django/db/backends/sqlite3/base.py

```diff
@@ -169,6 +169,25 @@ def get_new_connection(self, conn_params):
         conn.create_function("regexp", 2, _sqlite_regexp)
         conn.create_function("django_format_dtdelta", 3, _sqlite_format_dtdelta)
         conn.create_function("django_power", 2, _sqlite_power)
+        conn.create_function('ACOS', 1, math.acos)
+        conn.create_function('ASIN', 1, math.asin)
+        conn.create_function('ATAN', 1, math.atan)
+        conn.create_function('ATAN2', 2, lambda x, y: math.atan2(x, y))
+        conn.create_function('CEILING', 1, math.ceil)
+        conn.create_function('COS', 1, math.cos)
+        conn.create_function('COT', 1, lambda x: 1 / math.tan(x))
+        conn.create_function('DEGREES', 1, math.degrees)
+        conn.create_function('EXP', 1, math.exp)
+        conn.create_function('FLOOR', 1, math.floor)
+        conn.create_function('ln', 1, lambda x: math.log(x))
```

Comment:
Looks like this one got lowercased by mistake.

Reply:
Updated. Thanks!